lang/test/org/partiql/lang/eval/evaluatortestframework/AstEvaluatorTestAdapter.kt

```diff
@@ -65,45 +146,37 @@ class AstEvaluatorTestAdapter : EvaluatorTestAdapter {
 
     /**
      * Runs the give test case once with the specified [session].
-     *
-     * If non-null, [message] will be dumped to the console before test failure to aid in the identification
-     * and debugging of failed tests.
      */
     private fun privateRunEvaluatorTestCase(
         tc: EvaluatorTestCase,
         session: EvaluationSession,
-        message: String
+        note: String,
     ) {
-        val assertionMessage = message.let { "($it)" }
-
-        fun showTestCase() {
-            println(listOfNotNull(message, tc.groupName).joinToString(" : "))
-            println("Query under test  : ${tc.query}")
-            println("Expected value    : ${tc.expectedResult}")
-            println()
-        }
-
         val pipeline = tc.createPipeline()
 
-        val actualResult = try {
+        val actualExprValueResult = assertDoesNotThrow(
+            EvaluatorTestFailureReason.FAILED_TO_EVALUATE_QUERY,
+            { tc.testDetails(note) }
```

Comment:
I think in the case where the test case throws when it's expected to get a result, we should use the other `testDetails` function, which emits the error information (e.g. `actualErrorCode`, `actualErrorContext`, etc.)

Reply:
?? The other `testDetails` has a different receiver type, but I'll add `actualErrorCode`, `actualErrorContext` parameters to this function.

Reply:
My mistake. I had thought the extension functions had the same receiver type. As long as the error details are displayed somewhere, shouldn't be needed.